moveit_msgs/CollisionObject collision_object
---
bool success

